lib/api.js

```diff
@@ -122,8 +123,8 @@ PlexAPI.prototype._request = function _request(relativeUrl, method, parseRespons
             );
         }
 
-        if (response.statusCode !== 200) {
-            return deferred.reject(new Error('Plex Server didnt respond with status code 200, response code: ' + response.statusCode));
+        if (response.statusCode < 200 && response.statusCode > 299) {
```

Comment:
Oh my... What was I thinking.. I shouldn't make changes when I'm in a hurry I guess.. Pfff.